packages/nextra-theme/src/index.tsx

```diff
@@ -72,10 +178,11 @@ const mdxStyles: ThemeUICSSObject = {
   },
 }
 
+export const CodeInline = Code.Inline
```

Comment:
This could probably be removed and the only page that imports `CodeInline` could import GDS's `Code` instead.

Reply:
yeah, I already remove it next PR :) https://github.com/graphprotocol/docs/pull/732/files 
<img width="672" alt="image" src="https://github.com/user-attachments/assets/338196de-5333-4d5b-b936-9a975d852628">